src/app/plugins/dog.plugin.ts

```diff
@@ -7,7 +7,7 @@ export class DogPlugin extends Plugin {
   public name: string = 'Dog Plugin';
   public description: string = 'Generates pictures of doggos.';
   public usage: string =
-    'dog <(breed | subreed)(Optional)> | dog <breed> | dog <subBreed> | dog <random>';
+    'dog <(breed | subreed)(Optional)> | dog <listBreeds> | dog <listSubBreeds> <breed (Optional)>';
```

Comment:
when it's a keyword you dont put the `<>`:  so `dog listBreeds` not `dog <listBreeds>`